manifests/install.pp

```diff
@@ -9,10 +9,10 @@
 #
 class nsclient::install {
 
-  validate_string($nsclient::package_source_location)
-  validate_string($nsclient::package_source)
-  validate_string($nsclient::package_name)
-  validate_string($nsclient::install_path)
+  assert_type(String, $nsclient::package_source_location)
```

Comment:
Given init.pp is the entry point, these would have already generated a failure earlier, no?

Reply:
yes

Reply:
updated it